<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NguyenDuMap</class>
 <widget class="QMainWindow" name="NguyenDuMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="locale">
     <locale language="Vietnamese" country="Vietnam"/>
    </property>
    <property name="text">
     <string>Điểm bắt đầu (Màu đen)</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Bánh mỳ phúc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bánh tráng Phú Cường</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lẩu Nguyệt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ThaoYen cafe Nguyễn Du</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ngon garden</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bia hơi Hải Xòm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ưu đàm chay</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lotteria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bệnh viện y học cổ truyền TW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phủi quán</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đai Cát Foot Massage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Show Translations Circle K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kường Ngân Honda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phố đi bộ Trần Nhân Tông</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hoàng Hà Moblie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rạp xiếc TW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moji shop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chùa Thiên Quang</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nhà văn hóa học sinh sinh viên</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siêu thị Fujjimart Lê Duẩn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Karaoke sing</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="locale">
     <locale language="Vietnamese" country="Vietnam"/>
    </property>
    <property name="text">
     <string>Điểm đến (Màu đỏ)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Bánh mỳ phúc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bánh tráng Phú Cường</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lẩu Nguyệt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ThaoYen cafe Nguyễn Du</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ngon garden</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bia hơi Hải Xòm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ưu đàm chay</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lotteria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bệnh viện y học cổ truyền TW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phủi quán</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Đai Cát Foot Massage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Show Translations Circle K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kường Ngân Honda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phố đi bộ Trần Nhân Tông</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hoàng Hà Moblie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rạp xiếc TW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moji shop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chùa Thiên Quang</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nhà văn hóa học sinh sinh viên</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siêu thị Fujjimart Lê Duẩn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Karaoke sing</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-60</y>
      <width>921</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="test.qrc">:/newPrefix/phuongNguyenDu2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
